qiskit_nature/properties/second_quantization/electronic/integrals/electronic_integrals_utils/_electronic_integrals_utils.py

```diff
@@ -0,0 +1,118 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Utility functions to detect and transform the index-ordering convention of two-body integrals"""
+
+import numpy
+
+## These are the permutation symmetries satisfied by
+## a rank-4 tensor of real two-body integrals in chemists'
+## index order.
+## HJO is Molecular Electronic Structure Theory by Helgaker, Jørgensen, Olsen
+CHEM_INDEX_PERMUTATIONS = [
```

Comment:
I think it would be cleaner to make this an `Enum` rather than relying on the list index later on